ci/install_cuda_driver.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+CUDA_DRIVER_VERSION=$1
+
+echo "Installing CUDA driver version $CUDA_DRIVER_VERSION"
+apt-get -yqq install --no-install-recommends kmod wget
+wget -q https://us.download.nvidia.com/XFree86/Linux-x86_64/${CUDA_DRIVER_VERSION}/NVIDIA-Linux-x86_64-${CUDA_DRIVER_VERSION}.run
```

Comment:
This script seems to be specific for x86 arch, but the Dockerfile file is common to x86 and aarm64. You could pass an additional argument `ARCH` to the container build.
On  the other hand, I would really try to avoid this custom driver installation.

Reply:
Absolutely, this was just a test to see if this was the issue.